packages/create-app/src/cli.ts

```diff
@@ -122,6 +124,10 @@ export async function cli(args: string[] = process.argv): Promise<void> {
             array: true,
             describe:
               "List of client-side scopes to be used when creating a client",
+          })
+          .option("ontologyRid", {
+            type: "string",
+            describe: "RID of the Ontology your application is using",
```

Comment:
If we do change the prompt order then let's also change this option definition order. For the consistency of the terminology can you follow how application is defined? (call the option ontology and describe it as `Ontology resource identifier (rid)``

```
          .option("application", {
            type: "string",
            describe: "Application resource identifier (rid)",
          })
```

Reply:
Also if you don't mind keeping the README up to date (looks like we should put scopes in there too)
https://github.com/palantir/osdk-ts/tree/main/packages/create-app#options